week-1/L-functions-nested/exercise2.js

```diff
@@ -0,0 +1,17 @@
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+
+function mentors ( names) {
+    return names.toUpperCase(names);
+}
+function greetings (names){
+    return "HELLO "+ mentors(names)
```

Comment:
I was a little confused by your choice of variable name. How many people can I greet at once using this function?